scripts/presubmit-tests.sh

```diff
@@ -368,7 +368,9 @@ function main() {
       header "Documentation only PR, skipping running custom test"
       exit 0
     fi
-    ${TEST_TO_RUN} || failed=1
+    for test_to_run in ${TESTS_TO_RUN[@]}; do
+      ${test_to_run} || failed=1
```

Comment:
can you change "failed=1" to something like { failed=1; echo "--- FAIL: Test ${test_to_run}"; }?
Here and all uses of lone failed=1?

Reply:
I think it's a good idea. Updated and made a new Spyglass entry so that these messages could be highlighted